### PR TITLE
[SEMI-MODULAR] adding some stuffs that got removed during spell refactor

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Readded two spells, snap celeritas and magical stimpack, readded the bloodcrawl potion which replaced the mayhem potion from bubblegum and readded curator bundle that got removed because bloodcrawl potion was removed.

## How This Contributes To The Skyrat Roleplay Experience
what.

## Changelog

:cl:
add: Brought back magical stimpack and celeritas, curator is now able to choose ronin, again and bubblegum drop bloodcrawl potion instead of mayhem bottle.

/:cl:

### CODE REVIEW

File: modular_skyrat/modules/morewizardstuffs/spellbook.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//OFFENSIVE
+
+//DEFENSIVE
+
+//MOBILITY
+/datum/spellbook_entry/snapceleritas
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spellbook.dm]:
```suggestion
/datum/spellbook_entry/celeritas
```
Slightly better name

File: modular_skyrat/modules/morewizardstuffs/spellbook.dm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	name = "Celeritas"
+	desc = "With a snap of your finger, You will be able to swap places with someone!"
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spellbook.dm]:
```suggestion
	desc = "You will be able to swap places with someone just by snapping your fingers!"
```

File: modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/obj/item/bloodcrawlbottle
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm]:
```suggestion
/obj/item/bloodcrawl_bottle
```

File: modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+	name = "bloodlust in a bottle"
+	desc = "Drinking this will give you unimaginable powers... and mildly disgust you because of it's metallic taste."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "vial"
+
+/obj/item/bloodcrawlbottle/attack_self(mob/user)
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm]:
```suggestion
/obj/item/bloodcrawl_bottle/attack_self(mob/user)
```

File: modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	to_chat(user, span_notice("You drink the bottle's contents."))
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spells/bloodcrawl_potion.dm]:
```suggestion
	to_chat(user, span_notice("You drink the contents of [src]."))
```

File: modular_skyrat/modules/morewizardstuffs/spells/magical_stimpack.dm [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+	school = "transmutation"
+	cooldown_time = 10 SECONDS
+	cooldown_reduction_per_rank = 1.25 SECONDS
+	spell_requirements = NONE
+	invocation = "STIMULUS CHEQ'US"
+	invocation_type = INVOCATION_SHOUT
+
+/datum/action/cooldown/spell/stimpack/cast(mob/living/cast_on)
+	. = ..()
+	to_chat(cast_on, span_notice("Time appears to slow as your bodily functions rapidly speed up."))
```
Comment (tf-4) [inline modular_skyrat/modules/morewizardstuffs/spells/magical_stimpack.dm]:
```suggestion
	cast_on.balloon_alert(cast_on, "speeding up")
```

Comment (SarmentiCampbell) [reply modular_skyrat/modules/morewizardstuffs/spells/magical_stimpack.dm]: I know what it does but for whatever reason, it give a undefined proc error

Comment (tf-4) [reply modular_skyrat/modules/morewizardstuffs/spells/magical_stimpack.dm]: oops you're right, updated

**Review by tf-4 — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.

**Pull request merged**